mkt/extensions/serializers.py

```diff
@@ -60,13 +60,14 @@ def fake_object(self, data):
         obj = Extension(id=data['id'])
 
         # Create a fake ExtensionVersion for latest_public_version.
-        obj.latest_public_version = ExtensionVersion(
-            extension=obj,
-            id=data['latest_public_version']['id'],
-            created=data['latest_public_version']['created'],
-            size=data['latest_public_version'].get('size', 0),
-            status=STATUS_PUBLIC,
-            version=data['latest_public_version']['version'],)
+        if data['latest_public_version']:
```

Comment:
well, i could cache the results, and once the reviewer gets to the detail page, an XHR is fired to immediately refresh data.